File: app/src/main/java/ua/meugen/android/pdfcomposer/ui/activities/createnew/fragments/createnew/binding/CreateNewBinding.java
package ua.meugen.android.pdfcomposer.ui.activities.createnew.fragments.createnew.binding;

import ua.meugen.android.pdfcomposer.ui.activities.base.fragment.binding.Binding;
import ua.meugen.android.pdfcomposer.ui.activities.createnew.fragments.createnew.view.CreateNewView;

/**
 * @author meugen
 */
public interface CreateNewBinding extends Binding {

    void setupListeners(CreateNewView view);

    void displayContent(String imageUri, String text);
}

File: app/src/main/java/ua/meugen/android/pdfcomposer/ui/activities/createnew/fragments/createnew/binding/CreateNewBinding.kt
package ua.meugen.android.pdfcomposer.ui.activities.createnew.fragments.createnew.binding

import ua.meugen.android.pdfcomposer.ui.activities.base.fragment.binding.Binding
import ua.meugen.android.pdfcomposer.ui.activities.createnew.fragments.createnew.view.CreateNewView

/**
 * @author meugen
 */
interface CreateNewBinding : Binding {

    fun setupListeners(view: CreateNewView)

    fun displayContent(imageUri: String?, text: String?)
}
